### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by sdrave** [pymor/pymor#504]
Check that all instructions in the README file are up-to date.

**Comment by pmli** [comment]
It would be nice to change "Its main focus lies on the application of reduced basis methods".

I found some issues when using pip:
- `pip install pymor` doesn't work when I run it in a Python 3 virtualenv (raises `NameError: name 'execfile' is not defined`), but it does in Python 2.7
- running `pip install pymor[full]` says `pymor 0.4.2 does not provide the extra 'full'`

It might be good not to refer to Python 2.7 in
```
echo "$PYMOR_SOURCE_DIR/src" > $PYTHON_ROOT/lib/python2.7/site-packages/pymor.pth
```

**Comment by renefritze** [comment]
The last release on PyPi is 0.4.2 which is py2 only. The install extras ```[full], [full-nompi]``` etc are recent additions. Both issues will resolve themselves after the 0.5.0 upload.

**Comment by pmli** [comment]
Ok, cool. It would be nice to be explicit in the README about which Python versions are supported by which pyMOR versions (related to #325).

**Comment by renefritze** [comment]
Actually, the docker examples will have to wait until sometime after the release, because I first have to update the demo docker.

**Comment by renefritze** [comment]
I think this should all be included/changed  with #518
